src/haddock/libs/libhpc.py

```diff
@@ -320,6 +325,52 @@ def create_torque_header(
     return header
 
 
+def to_torque_time(time: int) -> str:
+    """Convert time in minutes to the form hh:mm:ss.
+
+    Parameters
+    ----------
+    time : int
+        Time in minutes.
+
+    Return
+    ------
+    hh_mm_ss : str
+        Time in the form for HH:MM:SS
+    """
+    hours = time // 60
+    remain_mins = time - (hours * 60)
+    # Convert to hh:mm:ss string
+    hh_mm_ss_l = [hours, remain_mins, 0]
+    # Make sure hours contain at least 2 characters
+    hh_mm_ss_str = (f'0{val}' if len(str(val)) == 1 else str(val)
+                    for val in hh_mm_ss_l)
+    # Join as single string
+    hh_mm_ss = ':'.join(hh_mm_ss_str)
+    return hh_mm_ss
```

Comment:
Why not use `datetime` here?

Reply:
`strftime` do not allow to write hours above 24, and format should be `hours:minutes:seconds`.
Also, to my knowledge, there is no obvious solution to format `datetime.timedelta`.